remediation-service/handler/problem_open_handler.go

```diff
@@ -32,26 +31,32 @@ func (eh *ProblemOpenEventHandler) HandleEvent() error {
 	// check if remediation should be performed
 	autoRemediate, err := eh.isRemediationEnabled()
 	if err != nil {
-		eh.KeptnHandler.Logger.Error(fmt.Sprintf("Failed to check if remediation is enabled: %s", err.Error()))
+		eh.KeptnHandler.Logger.Error(err.Error())
+		_ = eh.Remediation.sendRemediationFinishedEvent(keptn.RemediationStatusErrored, keptn.RemediationResultFailed, err.Error())
 		return err
 	}
 
 	if autoRemediate {
 		eh.KeptnHandler.Logger.Info(fmt.Sprintf("Remediation enabled for project %s in stage %s", problemEvent.Project, problemEvent.Stage))
 	} else {
-		eh.KeptnHandler.Logger.Info(fmt.Sprintf("Remediation disabled for project %s in stage %s", problemEvent.Project, problemEvent.Stage))
+		msg := fmt.Sprintf("Remediation disabled for service %s in project %s in stage %s", problemEvent.Service, problemEvent.Project, problemEvent.Stage)
+		eh.KeptnHandler.Logger.Info(msg)
+		_ = eh.Remediation.sendRemediationFinishedEvent(keptn.RemediationStatusErrored, keptn.RemediationResultFailed, msg)
 		return nil
 	}
 
 	// get remediation.yaml
 	resource, err := eh.Remediation.getRemediationFile()
 	if err != nil {
+		_ = eh.Remediation.sendRemediationFinishedEvent(keptn.RemediationStatusErrored, keptn.RemediationResultFailed, err.Error())
```

Comment:
Like above, should we log an error here?

Reply:
good catch - has been updated